problems/isOdd.js

```diff
@@ -7,8 +7,14 @@
 
 */
 
-function isOdd() {
-
+function isOdd(n) {
+  if (n % 2 === 1) {
+    return true;
+  } else if (typeof n !== "number") {
```

Comment:
I understand why you put this here but you don't need it because if you try to module 2 anything but a number it'll return NaN instead of 1 so it'll fall into the else category